vimeo-networking/src/main/java/com/vimeo/networking/model/uploadquota/Lifetime.java

```diff
@@ -22,40 +22,60 @@
  * SOFTWARE.
  */
 
-package com.vimeo.networking.model;
+package com.vimeo.networking.model.uploadquota;
 
 import com.google.gson.annotations.SerializedName;
 import com.vimeo.stag.UseStag;
 
+import org.jetbrains.annotations.Nullable;
+
 import java.io.Serializable;
 
 /**
- * Created by kylevenn on 8/19/15.
+ * Represents the lifetime quota limits imposed on the user. If there are no lifetime limits imposed on the user, then
+ * the fields will all be {@code null}.
  */
-@SuppressWarnings("unused")
 @UseStag
-public class Space implements Serializable {
+public class Lifetime implements Serializable {
 
-    private static final long serialVersionUID = -1985382617862372889L;
+    private static final long serialVersionUID = 5169142182182319211L;
 
+    @Nullable
     @SerializedName("free")
-    protected long mFree;
+    private Long mFree;
 
+    @Nullable
     @SerializedName("max")
-    protected long mMax;
+    private Long mMax;
 
+    @Nullable
     @SerializedName("used")
-    protected long mUsed;
+    private Long mUsed;
 
-    public long getFree() {
+    @Nullable
+    public Long getFree() {
         return mFree;
     }
 
-    public long getMax() {
+    void setFree(@Nullable Long free) {
+        mFree = free;
+    }
```

Comment:
(Optional) How do you feel about removing setters for new models? Our plan is to have immutable models in future iterations of networking, so we could start with new models.

That makes it so you'd have to find some way to clone on your own, but currently we're the largest consumer of this lib and I don't think we need to mutate this object. 

cc: @howardrigberg @msya

Reply:
I have made all the setters package private for this reason. No classes outside the `com.vimeo.networking.model.uploadquota` package can access the setter. We could remove the setters all-together, but then the fields need to be made package private or protected in order for stag to generate type adapters.

Reply:
I've forgotten how to java. Package private is 👌